pkg/integration/client_test.go

```diff
@@ -525,6 +527,58 @@ func TestDatabasesSwitching(t *testing.T) {
 	require.Nil(t, vi)
 }
 
+func TestDatabasesSwitchingWithInMemoryToken(t *testing.T) {
+	options := server.DefaultOptions().WithAuth(true)
+	bs := servertest.NewBufconnServer(options)
+
+	defer os.RemoveAll(options.Dir)
+	defer os.Remove(".state-")
+
+	bs.Start()
+	defer bs.Stop()
+
+	client, err := ic.NewImmuClient(ic.DefaultOptions().WithDialOptions(&[]grpc.DialOption{grpc.WithContextDialer(bs.Dialer), grpc.WithInsecure()}).WithTokenService(tokenservice.NewInmemoryTokenService()))
+	if err != nil {
```

Comment:
Minor: Why we're not using `require.NoError(t, err)` here? Is there some specific reason?